src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -150,23 +173,23 @@
             <Button x:Uid="KeyboardManager_RemapShortcutsButton"
                     Margin="{StaticResource SmallTopMargin}"
                     Style="{StaticResource ButtonRevealStyle}"
-                    Command="{Binding Path=EditShortcutCommand}"/>
+                    Command="{Binding Path=EditShortcutCommand}"
+                    IsEnabled="{x:Bind Path=ViewModel.Enabled, Mode=OneWay}"
+                    />
 
             <ListView x:Name="RemapShortcutsList"
                       extensions:ListViewExtensions.AlternateColor="{ThemeResource SystemControlBackgroundListLowBrush}"
-                      ItemsSource="{StaticResource dummyData}"
+                      ItemsSource="{Binding Path=RemapShortcuts}"
                       ItemTemplate="{StaticResource KeysListViewTemplate}"
                       BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}"
                       BorderThickness="1" 
                       CornerRadius="4"
-                      Width="350"
+                      MinWidth="350"
                       MaxHeight="200"
```

Comment:
![image](https://user-images.githubusercontent.com/32061677/80000137-9e495b80-8471-11ea-9330-b1adb2dc5db7.png)
When there are no remaps I think we should either have some min height so that it doesn't appear as a single line, or we should hide the listview whenever it is empty. What do you think would be better?
@saahmedm any suggestions for this?

Reply:
File new bug, low priority